app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/ColorCorrect.scala

```diff
@@ -145,7 +335,8 @@ object ColorCorrect {
         'alpha.as[Double].?, 'beta.as[Double].?,
         "min".as[Int].?, "max".as[Int].?,
         "saturation".as[Double].?,
-        'equalize.as[Boolean].?(false)
+        'equalize.as[Boolean].?(false),
+        'autoBalance.as[Boolean].?
```

Comment:
The type of `autoBalance` is `Option[Boolean]` so there's already a default value.